### PR TITLE
Feat/modularization

### PR DESCRIPTION
**Pull request by eleonov**

### CODE REVIEW

**Review by astorozhevsky — APPROVED** [review]

**Comment by eleonov** [comment]
@astorozhevsky please check after fixes

File: main.tf [hunk 1]
```diff
@@ -1,213 +1,52 @@
...
-}
-
-resource "google_project_service" "enable_compute_api" {
-  project = var.project_id
-  service = "compute.googleapis.com"
-}
-
-resource "google_project_service" "enable_container_api" {
+resource "google_compute_address" "ingress_ip_address" {
+  name    = "${var.project_name}-${var.environment_name}-nginx"
```
Comment (astorozhevsky) [inline main.tf]: I suggest renaming this to `${var.project_name}-${var.environment_name}-nginx-ingress-controller`

File: modules/cloud-sql/output.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+output "generated_user_password" {
+  value       = module.postgresql.generated_user_password
+  description = "The password of the defaule user to access database"
```
Comment (astorozhevsky) [inline modules/cloud-sql/output.tf]:
```suggestion
  description = "The password of the default user to access database"
```

File: modules/cloud-sql/variables.tf [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  type        = string
+}
+
+variable "cluster_region" {
+  description = "The region of the cluster"
+  type        = string
+}
+
+variable "cluster_network" {
+  description = "The region of the cluster"
```
Comment (astorozhevsky) [inline modules/cloud-sql/variables.tf]: the same description as above

**Review by astorozhevsky — CHANGES_REQUESTED** [review]

**Review by astorozhevsky — APPROVED** [review]

**Pull request merged**